src/debug/debugSupport.ts

```diff
@@ -16,6 +16,12 @@
 
 import * as sourceMap from './sourceMap';
 import { getFromENV } from '../helper';
+import { BrowserContextBase } from '../browserContext';
+import { Frame } from '../frames';
+import { Events } from '../events';
+import { Page } from '../page';
+import { parseSelector } from '../selectors';
```

Comment:
Doesn't injected script know how to parse selectors?

Reply:
Nope, we parse them in Node js to handle main/utility worlds and to provide better error messages.